### ISSUE TITLE
Add "normalized_title" property

### ISSUE THREAD

**Issue by gereons** [zaroth/netrunner-cards-json#14]
Suggestion: add a "normalized_title" property for those few card names that contain diacritical marks, like e.g. Wünderkind and Pālanā. (These would be "Wunderkind" and "Palana", respectively). This would enable environments without decent support for removing diacriticals to be able to better search for such names.

**Comment by Alsciende** [comment]
That property would only accept ASCII values, correct?

**Comment by gereons** [comment]
Exactly. If I'm not mistaken, there are currently 9 cards needing this treatment:

Doppelganger
Palana Foods: Sustainable Growth
Palana Agroplex
Chaos Theory: Wunderkind
Deja Vu
Shi.Kyu
Tori Hanzo
Expose
Draco

**Comment by zaroth** [comment]
Not that I'm against the idea of helping normalization, but card titles aren't the only ones with diacritics. The other obvious candidates are `illustrator` and `text`. I do get that the value is lower here, though, since they are probably searched an order of magnitude less often.

**Comment by Alsciende** [comment]
Could the addition of normalized_title be done by the `validate.py`? Compute the normalized title, and if it is different from the title, set the property (overwriting it if needed)?

**Comment by zaroth** [comment]
Sure it could, for example by using https://pypi.python.org/pypi/Unidecode. My question was more along the lines if we should stop at normalizing titles, or consider doing it to other fields as well.

**Comment by Alsciende** [comment]
I guess you're right, we could do the same thing for text. I don't know about illustrators.

Maybe we should call the parameter "title_normalized" instead, for sorting purpose.

**Comment by zaroth** [comment]
Ok, so `title_normalized` and `text_normalized` it is. The validator needs to be updated.

Regarding `text_normalized` - it should be pure ASCII, so long dashes after trace messages will be translated to minus signs from ASCII range too.

**Comment by Alsciende** [comment]
Good point for the long dash. Agreed.

**Comment by gereons** [comment]
Sounds good, although I can't immediately find important diacritcs in `text` (with the exception of self-referential text like on Draco) that people would ususally want to search for.

**Comment by zaroth** [comment]
Well, self-referential was mostly my thought. Also considering naming the properties `foo_ascii`, instead of `foo_normalized`. Straight to the point.

**Comment by Alsciende** [comment]
Agreed

**Comment by Saintis** [comment]
Any progress on this?
